src/frontend/komponenter/Oppgavebenk/OppgavelisteSaksbehandler.tsx

```diff
@@ -62,9 +66,12 @@ const OppgavelisteSaksbehandler: React.FunctionComponent<IOppgavelisteSaksbehand
         </div>
     ) : (
         <div className={'kolonne'}>
-            <Normaltekst>Ikke tildelt</Normaltekst>
+            <Normaltekst className="tilordnet-ressurs-tekst">Ikke tildelt</Normaltekst>
```

Comment:
Vi har egentlig gått over fra å bruke less til styled components. Tror du kan skrive
```
const StyledNormaltekst = styled(Normaltekst)`
    width: 5rem;
    margin-right: 2.5rem;
    text-align: left;
`;
```

også kan du byttet ut dette med

```suggestion
            <StyledNormaltekst>Ikke tildelt</StyledNormaltekst>
```

Reply:
Ahh sant det! Fikser 👍